client/src/components/desktop/HomeDesktop.js

```diff
@@ -1,34 +1,26 @@
 import React from 'react';
-import { Textfit } from 'react-textfit';
 import { Col, Row, Carousel } from 'antd';
 
 import testimonials from '../../data/testimonials';
 
 export const HomeBlock2Desktop = () => {
   return (
     <Row className="home-block-2" type="flex" justify="center" align="middle">
-      <Col className="home-block-2__left" span={6}>
-        <Row
-          className="home-block-2__left__text"
-          type="flex"
-          justify="start"
-          align="middle"
-        >
-          <Col span={20}>
-            <Textfit mode="single">
```

Comment:
why did we have the textfit in the first place?

Reply:
Honestly I have no idea